yeast-markdown-parser/src/__tests__/resources/images.ts

```diff
@@ -455,55 +416,9 @@ export const IMAGE_LINKS_AST = {
 				{
 					type: 'listitem',
 					level: 0,
-					children: [
-						{
-							type: 'bold',
-							children: [
-								{
-									text: 'Documentation',
-								},
-							],
-						},
-						{
-							text: ' ',
-						},
-						{
-							type: 'link',
-							children: [
-								{
-									text: 'https://developer.genesys.cloud/devapps/sdk/docexplorer/purecloudjava/',
-								},
-							],
-							href: 'https://developer.genesys.cloud/devapps/sdk/docexplorer/purecloudjava/',
-						},
-					],
-				},
-				{
-					type: 'listitem',
-					level: 0,
-					children: [
-						{
-							type: 'bold',
-							children: [
-								{
-									text: 'Source',
-								},
-							],
-						},
-						{
-							text: ' ',
-						},
-						{
-							type: 'link',
-							children: [
-								{
-									text: 'https://github.com/MyPureCloud/platform-client-sdk-java',
-								},
-							],
-							href: 'https://github.com/MyPureCloud/platform-client-sdk-java',
-						},
-					],
+					children: [{ text: '**Documentation** https://developer.genesys.cloud/devapps/sdk/docexplorer/purecloudjava/' }],
```

Comment:
This wasn't intended; not sure why this changed. Most of the rest of the changes to this AST were just JSON formatting but noticed this in the diff.